bin/fixmystreet.com/one-off-status-update

```diff
@@ -78,13 +78,26 @@ if (!$commit) {
     say "*** DRY RUN ***";
 }
 
-my %alloy_to_fms_map = (
+my %enq_to_fms_map = (
+    'Issued to Inspector' => 'investigating',
+    'In Progress' => 'investigating',
+    'Completed' => 'closed',
```

Comment:
Do they close rather than mark as fixed?

Reply:
Completed is the state for an inspection essentially. It doesn't get marked as fixed unless a defect is raised. And in theory there should be a reason for closure set when an inspection is completed which will override this, but we need a default.